### DISCUSSION TITLE
"darktable.app" is damaged and can't be opened. You should move it to the Bin.

### DISCUSSION THREAD

**Discussion by cptharris** [Homebrew/discussions#4778] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.10
ORIGIN: https://github.com/Homebrew/brew
HEAD: 12c8778af976269344f3c0b914243e178fc4ffbd
Last commit: 3 days ago
Core tap JSON: 09 Sep 20:45 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.5.2-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: 14.3.1
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

After installing darktable, attempting to launch the app fails with ```"darktable.app" is damaged and can't be opened. You should move it to the Bin.```

This is documented by the darktable team (see [this explanation for their solution](https://github.com/darktable-org/darktable/issues/14746#issuecomment-1612305326)). The [darktable website provides instructions](https://www.darktable.org/install/#macos) for avoiding this error, but a user installing from homebrew might never see them.

I propose either running the command that dequarantines darktable or adding a post-install note so users remember to do it themselves. Not sure how homebrew handles this sort of thing.

Thanks!

### Relevant casks

darktable

**Comment by SMillerDev** [comment]
Quarantine is a security feature we go to great lengths to enable. The real fix is for upstream to sign their app.
